lib/Yield/Pipes/CallGraphPipes.cpp

```diff
@@ -68,7 +69,34 @@ ProcessCallGraphPipe::getContract() const {
 void YieldCallGraphPipe::run(pipeline::Context &Context,
                              const FileContainer &Input,
                              FileContainer &Output) {
-  revng_abort("WIP");
+  // Access the model
+  const auto &Model = revng::pipes::getModelFromContext(Context);
+
+  // Open the input file.
+  auto MaybeInputPath = Input.path();
+  revng_assert(MaybeInputPath.has_value());
+  auto MaybeBuffer = llvm::MemoryBuffer::getFile(MaybeInputPath.value());
+  revng_assert(MaybeBuffer);
+  llvm::yaml::Input YAMLInput(**MaybeBuffer);
```

Comment:
Aaah, we need a `YamlizableContainer`. It's on Massimo's TODO list.
Add a `// TODO` to adopt it.

Reply:
All of my pipes need it, actually, so don't worry, I won't forget. This one of those things (together with pipeline configuration support and stuff) I can't wait to get added.